src/js/controller/captions.js

```diff
@@ -84,10 +85,11 @@ const Captions = function(_model) {
         track.data = vttCues;
     }
 
-    function _addTrack(track) {
+    function _addTrack(track, isSideload) {
```

Comment:
we already have an attribute created for a different fix for distinguishing sideloaded captions, can you use this? https://github.com/jwplayer/jwplayer/pull/3570/files#diff-516afba75eaadc497dca75d8c83b0fe6R557

Reply:
Updated, thank you!